tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -184,4 +184,29 @@ object CompletionMatchSuite extends BaseCompletionSuite {
       "2.11" -> "match"
     )
   )
+
+  // https://github.com/scalameta/metals/issues/1253
+  checkEdit(
+    "exhaustive-fully-qualify".expectedToFail,
```

Comment:
here's an example. You could technically also do 

```suggestion
    "exhaustive-fully-qualify".tag(Tag.ExpectedFailure),
```

but since we'll reasonably have a few tags, I opted for a utility "DSL" method

Reply:
I think it's fine to have a method here to ease discovery.